.github/workflows/flake8.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       run: pip install flake8
 
     - name: Run Flake8
-      run: flake8 pyiddidf
+      run: flake8 energyplus_iddidf
```

Comment:
Renamed source directory to energyplus_iddidf.  Trying to unify all the imports to `import energyplus*` for the best user experience.